docs/Advanced-Usage/Chip-Communication.rst

```diff
@@ -0,0 +1,147 @@
+.. _chip-communication:
+
+Communicating with the Chip/DUT
+===============================
+
+What good is a chip/DUT if it can't communicate with the outside world? There are two types of designs that can be
```

Comment:
this is stylistic, but the question doesn't really seem to flow into the rest of the text